contracts/oracle/ChainlinkOracleWrapper.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+import "./IOracle.sol";
+import "../refs/CoreRef.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";
+
+/// @title Chainlink oracle wrapper
+/// @author eswak
+/// @notice Reads a Chainlink oracle value & wrap it under the standard Fei oracle interface
+contract ChainlinkOracleWrapper is IOracle, CoreRef {
+    using Decimal for Decimal.D256;
+    using SafeMath for uint256;
+
+    /// @notice the referenced chainlink oracle
+    AggregatorV3Interface public chainlinkOracle;
+    uint256 public oracleDecimalsNormalizer;
+
+    /// @notice ChainlinkOracleWrapper constructor
+    /// @param _core Fei Core for reference
+    /// @param _chainlinkOracle reference to the target Chainlink oracle
+    constructor(
+        address _core,
+        address _chainlinkOracle
+    ) public CoreRef(_core) {
+        chainlinkOracle = AggregatorV3Interface(_chainlinkOracle);
+
+        _init();
+    }
+
+    function _init() internal {
+        uint8 oracleDecimals = chainlinkOracle.decimals();
+        oracleDecimalsNormalizer = 10 ** uint256(oracleDecimals);
+    }
+
+    /// @notice updates the oracle price
+    /// @return true if oracle is updated and false if unchanged
+    function update() external override whenNotPaused returns (bool) {
+        return false;
```

Comment:
perhaps this should return true for clarity so that callers don't assume something is wrong when updating this oracle per 4.13. The false return value isn't meant to trigger reversions anywhere else so no worried on requiring true in other places in the code.

Reply:
for semantics I'd like to leave it to return `false`, since it does not trigger an update on the Chainlink oracle.

I added an isOutdated() check in the swap() function, and only then I call update().